client/van_router_create.go

```diff
@@ -19,7 +17,33 @@ import (
 	"github.com/skupperproject/skupper/pkg/utils/configs"
 )
 
-func GetVanControllerSpec(options types.VanRouterCreateOptions, van *types.VanRouterSpec, transport *appsv1.Deployment) {
+func OauthProxyContainer(serviceAccount string) *corev1.Container {
+	return &corev1.Container{
+		Image: "openshift/oauth-proxy:latest",
+		Name:  "oauth-proxy",
+		Args: []string{
+			"--https-address=:8443",
+			"--provider=openshift",
+			"--openshift-service-account=" + serviceAccount,
+			"--upstream=http://localhost:8888",
```

Comment:
Use types.ConsoleOpenShiftServicePort here?

Reply:
Actually, this is used for both router console and skupper controller console, so unless those always use the same ports, might need to pass ports in?

Reply:
Good suggestion. I will update. In reviewing skupper-cli, it looked like router console and skupper controller console are using the same ports. Agree there may be cases to separate.